src/openrct2/world/Footpath.cpp

```diff
@@ -1502,7 +1502,7 @@ void PathElement::SetIsBlockedByVehicle(bool isBlocked)
     }
 }
 
-uint8_t PathElement::GetStationIndex() const
+::StationIndex PathElement::GetStationIndex() const
```

Comment:
why the extra `::`

Reply:
I believe it's in order to differentiate between the field and the typedef. At least that why I had to do so for `SetStationIndex()` - I assumed that the same would apply for `GetStationIndex()`.

Reply:
oh yeah right you are.